<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T3_AF_map</class>
 <widget class="QDialog" name="T3_AF_map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white</string>
  </property>
  <widget class="QLabel" name="_dateTimeLabel_">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>150</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>_dateTImeLabel_</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
   </property>
  </widget>
  <widget class="QPushButton" name="_exitPushBtn_">
   <property name="geometry">
    <rect>
     <x>745</x>
     <y>8</y>
     <width>30</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>_exitPushBtn_</string>
   </property>
  </widget>
  <widget class="QLabel" name="_showConnectStatus_">
   <property name="geometry">
    <rect>
     <x>646</x>
     <y>48</y>
     <width>98</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="_clear">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>223</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/clearPath.png);color:white</string>
   </property>
   <property name="text">
    <string>清除路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="_videoLabel_">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>37</y>
     <width>221</width>
     <height>166</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="_modePushBtn_">
   <property name="geometry">
    <rect>
     <x>646</x>
     <y>93</y>
     <width>98</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;border-width:0;border-style:outset;color:white</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="_stopPushBtn_">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>315</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/mainWindow_stop.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="_videoSquareLabel_">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/Pictures/map_videoSquare.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="_battery_">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>420</y>
     <width>35</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(7, 221, 225)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="_battIMG_">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>95</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
